app/models/order.rb

```diff
@@ -4,8 +4,15 @@ class Order < ApplicationRecord
   has_many :items, through: :order_items
   has_many :order_items, dependent: :destroy
 
+  def order_total
+    if status == 'open'
+      order_items.map { |oi| oi.item[:price] * oi[:quantity].to_f }.reduce(0.0, :+)
```

Comment:
consider creating an `order_item#total` method that does the math on each item for you